### PR TITLE
doc: fix typo in package metrics

### PR DESCRIPTION
**Pull request by halfcrazy**
**What this PR does / why we need it**:
fix typo in package metrics

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by php-coder** [comment]
@halfcrazy Please, next time, fill the release note field, so it will have an appropriate label from the beginning.

**Comment by halfcrazy** [comment]
@php-coder Sorry, I have updated all my release note field.

File: staging/src/k8s.io/metrics/IMPLEMENTATIONS.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 ## Custom Metrics API
 
-***NB: None of the below implemenations are officially part of Kubernetes.
+***NB: None of the below implementations are officially part of Kubernetes.
```
Comment (nikhita) [inline staging/src/k8s.io/metrics/IMPLEMENTATIONS.md]: This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464 - https://github.com/kubernetes/kubernetes/pull/59464/files#diff-1c26f9847072f97a5b83d3a5f2dbc012. Closing.

/close

**Pull request closed without merging**